src/openms/source/DATASTRUCTURES/SuffixArraySeqan.cpp

```diff
@@ -171,7 +171,7 @@ namespace OpenMS
   //destructor
   SuffixArraySeqan::~SuffixArraySeqan()
   {
-
+    delete it_;
```

Comment:
mhh.. looking at the code, there seems to be no reason to have _it in the first place.
Its used like a local variable and is always dereferenced.
But there are multiple assignments with new(), which should also be prominent mem-leaks.
'index_' however seems not to be initialized all the time...

can we thrash this whole class?!